src/colvarbias_abf.h

```diff
@@ -202,23 +202,15 @@ class colvarbias_abf : public colvarbias {
   /// Alternately we could overload the init_dependencies() function
   virtual int init_dependencies();
 
-  enum features_bias_abf {
-    /// Start after generic cvb features
-    /// There is at least one ext-Lagrangian colvar -> run eABF
-    f_cvb_abf_extended = f_cvb_ntot,
-    /// Total number of features for an ABF bias
-    f_cvb_abf_ntot
-  };
-
   /// \brief Implementation of the feature list for colvarbias_abf
   static std::vector<feature *> cvb_abf_features;
```

Comment:
Sorry, I may be missing something, but is this still needed if the features are all stored in the base class?

Reply:
Currently there is a specific dependency:
```
require_feature_self(f_cvb_history_dependent, f_cvb_get_total_force);
```
But I can bypass the dependency system for this specific case.

Reply:
Ok, thanks, I get it now (the vector is a static member). Makes sense, I just hope this doesn't bite us back in the future with a subtle bug.

Reply:
Done.